rest-service/manager_rest/storage/resource_models.py

```diff
@@ -349,7 +349,18 @@ class Deployment(CreatedAtMixin, SQLResourceBase):
     workflows = db.Column(db.PickleType(
         protocol=2, comparator=lambda *a: False))
     runtime_only_evaluation = db.Column(db.Boolean, default=False)
-
+    installation_status = db.Column(db.Enum(
+        DeploymentState.ACTIVE,
+        DeploymentState.INACTIVE,
+        name="installation_status"
+    ))
+    deployment_status = db.Column(db.Enum(
+            DeploymentState.GOOD,
+            DeploymentState.IN_PROGRESS,
+            DeploymentState.REQUIRE_ATTENTION,
```

Comment:
If the database enum does change, the constant probably should too

Reply:
check the above comment regarding `required_attension`